java-native/src/test/java/com/baeldung/jvm/JVMBitVersionUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.jvm;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.sun.jna.Platform;
+
+public class JVMBitVersionUnitTest {
+
+    private static JVMBitVersion jvmVersion;
+
+    @Before
+    public void setup() {
+        jvmVersion = new JVMBitVersion();
+    }
+
+    @Test
+    public void testGetUsingSystemClass() {
```

Comment:
Use the givenX_whenY_thenZ naming convention for the test methods (the givenX part is optional).